Extension/src/LanguageServer/ui.ts

```diff
@@ -432,9 +540,9 @@ export class UI {
 
     public dispose(): void {
         this.configStatusBarItem.dispose();
-        this.browseEngineStatusBarItem.dispose();
-        this.intelliSenseStatusBarItem.dispose();
-        this.referencesStatusBarItem.dispose();
+        if (this.browseEngineStatusBarItem) {this.browseEngineStatusBarItem.dispose(); }
```

Comment:
Can you use `this.browseEngineStatusBarItem?.dispose();` instead?

Reply:
That gives a syntax error since `?` is meant for expressions apparently

Reply:
Can you do `void this.browseEngineStatusBarItem?.dispose();`?  The 'void' might convert it to an expression that the linter won't complain about.

Reply:
Oh...I'm okay with either approach....but if you're sticking with the `if (){` there should be space after the `{` -- I think running the formatter would fix it.